### DISCUSSION TITLE
No support for Apple Silicon Macs as host machines, right?

### DISCUSSION THREAD

**Discussion by Emreler1** [LizardByte/community#289] [Sunshine] [answered]
Sorry if this has already been covered but I could not be sure about this. I am using an M1 Macbook Air as both a work and gaming (!) pc. So I wanted to understand whether I can use Sunshine on it in order to cast games to my living room TV. It seems like since GPU of Apple Silicon computers are not listed in the supported GPUs list, I cannot do this. Can you please verify?

Thanks

**Comment by ReenigneArcher** [comment, marked as answer]
v0.19.1 should work, there's a bug in newer versions though.

P.S. there is no gamepad support for macOS at this time.

**Reply by Emreler1** [reply]
Thanks but dangit! That release does not have a mac version (*.dmg file)

**Reply by ReenigneArcher** [reply]
Only macports is supported as the moment. Any dmg/pkg should be avoided in my opinion, at this time.

**Reply by Emreler1** [reply]
I am a beginner. So please bear with me. What is a macport? Which one should I install from the v0.19.1 release?

**Reply by ReenigneArcher** [reply]
This is why we have docs. https://docs.lizardbyte.dev/projects/sunshine/en/latest/about/installation.html#portfile
